### PR TITLE
Restructure readme

### PR DESCRIPTION
**Pull request by matt-frey**
This PR puts individual sections into their own markdown files in order to improve clarity.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,300 +1,12 @@
-# Welcome to the EPIC workshop in Trieste!
+# Welcome to the EPIC workshop!
 
 This repository serves as a tutorial to show you how to prepare, run and analyse a moist bubble
 simulation using the [Elliptical Parcel-In-Cell (EPIC)](https://doi.org/10.1016/j.jcpx.2023.100136)
-method. During the course of this workshop (July 8th 2024), we provide you temporary access to the
+method. During the course of this workshop, we provide you temporary access to the
 Cirrus cluster, a Tier-2 system of the EPSRC. All information how to access Cirrus is provided below.
```
Comment (RuiApostolo) [inline README.md:6]: The phrasing we ask to be used is:

"Cirrus UK National Tier-2 HPC Service at EPCC (http://www.cirrus.ac.uk) funded by the University of Edinburgh and EPSRC (EP/P020267/1)"

File: sites/cluster_access.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Accessing the Cirrus cluster
+
+Detailed information about the Cirrus cluster can be found in the [Cirrus website](https://www.cirrus.ac.uk/about/hardware.html) and in [this repo](lessons/01-Cirrus.md).
+
+#### Summary:
+  * Create SSH key-pair
+  * Create SAFE account
+  * Accept invite to `d185`, you will need to add the public part of an SSH key-pair to the `d185` machine account in SAFE when accepting the invite
```
Comment (RuiApostolo) [inline sites/cluster_access.md:8]: All `d185` need to be replaced with `tc066`

File: README.md [hunk 3]
```diff
@@ -1,300 +1,12 @@
...
-
-plt.savefig('xz-buoyancy-cross_section.png', bbox_inches='tight', dpi=200)
-plt.close()
-```
-
-![Cross sections through the centre of the y-axis.](figures/xz-buoyancy-cross_section.png
-"Cross sections through the centre of the y-axis.")
+- [Accessing the Cirrus cluster](sites/cluster_access.md)
+- [Setting up the environment](sites/setup_environment.md)
+- [Moist bubble test case](sites/moist_bubble.md)
```
Comment (RuiApostolo) [inline README.md:12]: I would maybe number the files so they show up "in order", like:
```
01-cluster_access.md
02-setup_environment.md
03-moist_bubble.md
```

File: sites/moist_bubble.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+| plot_intersected_ellipses.py  | Create a cross section plot where all intersection ellipses are displayed. |
+| plot_parcel_history.py        | Create a scatter plot with the original and final parcel height. |
+
+These scripts allow you to explore Eulerian and Lagrangian simulation data.
+Information how to use these scripts is printed when running with the flag `--help`, e.g. `plot_cross_sections.py --help`.
+
+> [!TIP]
+> We suggest that you append the plotting directory to the `$PATH` environment variable with
+> ```bash
+> export PATH=$PATH:/work/d185/d185/$USER/workshop-trieste/plotting
```
Comment (RuiApostolo) [inline sites/moist_bubble.md:115]: `d185` again

File: sites/setup_environment.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+- [How to load the Python virtual environment](#how-to-load-the-python-virtual-environment)
+
+
+## Obtaining all resources
+Before you start, you must clone this repository to your working directory on Cirrus.
+You can do this by using the command:
+
+```bash
+cd /work/d185/d185/$USER
+git clone https://github.com/EPIC-model/workshop-trieste.git
```
Comment (RuiApostolo) [inline sites/setup_environment.md:13]: `d185`

and do we want to change the repository name?

**Review by RuiApostolo — CHANGES_REQUESTED** [review]

File: sites/setup_environment.md (hunk 5 above)
Comment (matt-frey) [reply sites/setup_environment.md]: Note: I have already changed the repo name; though I did not do in the markdown files. Done now.

File: README.md (hunk 3 above)
Comment (matt-frey) [reply README.md]: OK. Done.

**Review by RuiApostolo — APPROVED** [review]
Looks good. I'll see if I can make the account creation/login a bit more readable

**Pull request merged**